plugins/E1.31/e131plugin.cpp

```diff
@@ -173,8 +173,7 @@ bool E131Plugin::openOutput(quint32 output, quint32 universe)
         E131Controller *controller = new E131Controller(m_IOmapping.at(output).iface,
                                                         m_IOmapping.at(output).address,
                                                         output, this);
-        connect(controller, SIGNAL(valueChanged(quint32,quint32,quint32,uchar)),
-                this, SIGNAL(valueChanged(quint32,quint32,quint32,uchar)));
+        connect(controller, &E131Controller::valueChanged, this, [this](quint32 _t1, quint32 _t2, quint32 _t3, uchar _t4){ valueChanged(_t1, _t2, _t3, _t4); });
```

Comment:
This lambda is necessary because the fuction `E131Plugin::valueChanged` has 5 parameters while the signal only has 4. The fifth one is an optional parameter, but the compiler doesn't understand that.